src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeAlwaysOnTop.xaml

```diff
@@ -4,31 +4,26 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:custom="using:Microsoft.PowerToys.Settings.UI.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:local="using:Microsoft.PowerToys.Settings.UI.OOBE.Views"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:toolkitcontrols="using:CommunityToolkit.WinUI.UI.Controls"
     mc:Ignorable="d">
 
     <custom:OOBEPageControl x:Uid="Oobe_AlwaysOnTop" HeroImage="ms-appx:///Assets/Settings/Modules/OOBE/AlwaysOnTop.png">
         <custom:OOBEPageControl.PageContent>
-            <StackPanel Orientation="Vertical">
-                <TextBlock x:Uid="Oobe_HowToUse" Style="{ThemeResource OobeSubtitleStyle}" />
+            <StackPanel Orientation="Vertical" Spacing="12">
```

Comment:
Top spacing is gone now? Also the subtitles are increased in size. 

New vs. old
![image](https://github.com/microsoft/PowerToys/assets/9866362/a04b8030-d2ee-42db-8f1e-0334a8516bd7)

Reply:
Spacing at the top needs to be reverted, subtitles as well.. this was done per design spec so we don't want to revert from that. There wasn't anything broken, right :)?

Reply:
Is that design spec public? The current style looks the same as headers for settings (groups) but this isn't settings.

Reply:
I don't think it is, we were following an internal draft of an OOBE page. But since we didn't get any feedback on this specifically (from users or internal teams) maybe we should leave this as is? We are thinking of a bigger overhaul of the entire OOBE experience, will post an update in the issue once I have that

Reply:
Are you okay with using this style though? Since I can't know the design spec you are referring to, and it does add a bit of hierarchy.

Reply:
The spacing was already providing the hierarchy - and these titles are not the primary to convery, especially since they are the same on every page. Therefore they wouldn't need additional emphasis.

The additional spacing between the buttons at the bottom and the text has changed now as well? Might want to increase that, see earlier comment.

Reply:
I've reset the visual to what it was.